lib/pages/people/view/courses_list_view.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
```

Comment:
This is a big no-no, firebase code should never be in a view component (see the workshop - search for keyword "BLoC"). Additionally, there's already a widget for this, just use the existing one: https://github.com/student-hub/acs-upb-mobile/blob/master/lib/pages/classes/view/classes_page.dart#L209

Reply:
The firebase code is only to mock the final look of the page. 
We will add a function in `PersonProvider` to get all the classes by lecturer similar to this https://github.com/student-hub/acs-upb-mobile/blob/master/lib/pages/people/service/person_provider.dart#L56-L75

Thanks for pointing out to `ClassList`

Reply:
That file is no longer used, now we use [classes_card](https://github.com/student-hub/acs-upb-mobile/blob/396dff90559907416f9ea31238fb87d37c747e59/lib/pages/people/view/classes_card.dart).

Reply:
now it uses `ClassList` https://github.com/student-hub/acs-upb-mobile/pull/252/commits/9c4fc131c40f2e63c474bc2c9adbd0e1bad643c0